pkg-manager/link-bins/src/isFileOwnedByUser.ts

```diff
@@ -0,0 +1,7 @@
+import { promises as fs } from 'fs'
+
+export async function isFileOwnedByUser (filePath: string): Promise<boolean> {
```

Comment:
TODO: what about root user? We don't want to skip fixing bins if the target file is not owned by root user.